index.html

```diff
@@ -73,22 +73,39 @@
 			h1 {
 				font-size: 140%;
 			}
-		</style>
+
+            .embed-container {
+                position: relative;
+                height: 0;
+                overflow: hidden;
+            }
+
+            .ratio-audience {
+                padding-bottom: 48.33333333333333%;
+            }
+
+            .embed-container img {
+                position: absolute;
+                top: 0;
+                left: 0;
+                width:100%;
+            }
+        </style>
 		
 		<link rel="stylesheet" type="text/css" href="css/animate.css"/>
 	</head>
-	<body onload="animate()">
+	<body>
```

Comment:
actually, my change wasn't about syntactical ugliness but about "for some freakish reason the animation doesn't work if you wait for the onload event".

And yes, I see your point about images not being loaded. I have another idea to try; will update the pull request if it works out